src/assets/styles/components/_footer.scss

```diff
@@ -23,7 +23,10 @@
       margin-bottom: 12px;
     }
   }
-
+  .opp-badge img {
+    width: 139px;
+    top: -16px;
```

Comment:
why -16px? isn't this aligned with other cols?

Reply:
The image has paddings on top and bottom, and without the top: -16px doesn't look aligned with the other widgets on the footer. Let me know any suggestions.